games/space-invaders/Space Invaders [David Winter].json

```diff
@@ -0,0 +1,108 @@
+{
+  "title": "Space Invaders",
+  "authors": [
+    {
+      "id": "david-winter",
+      "name": "David Winter"
+    }
+  ],
+  "release": "1978",
```

Comment:
Most of the apps don't have any metadata and for some we only know the year of release. I propose to relax the requirement so that this field may contain year, year-month or year-month-day for homebrew.

Reply:
Yep, ISO 8601 allows dropping the day, or both the day and month, while still being a valid date. So we should probably just follow the ISO 8601 standard here, allowing stuff like "1978", "1978-05", or "1978-05-25".

Reply:
Great.